howtos/markdown-and-mdx-features.md

```diff
@@ -1,4 +1,4 @@
-# Markdown and MDX Features
```

Comment:
Thank you for catching a style guide issue!

Reply:
😅 @christinaausley caught it in the original PR and I just copied it over 😅